### PR TITLE
Enable sims in the CI

### PR DESCRIPTION
**Pull request by chetanyagoyal**
- Edits yaml config file for temp-sense tests (adds job for simulation test)
- Updates parse report file in scripts folder to add sim check
- Misc. cleanup

### CODE REVIEW

File: .github/scripts/parse_rpt.py [hunk 1]
```diff
@@ -31,3 +32,33 @@
         raise ValueError("LVS failed!")
     else:
         print("LVS is clean!")
+
+if len(sys.argv) > 1 and sys.argv[1] == "sky130hd_temp_full":
+    result_filename = "work/prePEX_sim_result" # might remove from check or need to implement different logic later if sim outputs change due to downstream changes
+    sim_state_filename = "work/sim_state_file.txt"
+
+    with open(result_filename) as f2, open("../../../.github/scripts/expected_sim_outputs/prePEX_sim_result.txt") as f1:
```
Comment (saicharan0112) [inline .github/scripts/parse_rpt.py:40]: the `...../expected_sim_outputs/prePEX_sim_result.txt` is not added yet? Is this PR still in progress?

**Review by saicharan0112 — CHANGES_REQUESTED** [review]

File: .github/scripts/parse_rpt.py (hunk 1 above)
Comment (chetanyagoyal) [reply .github/scripts/parse_rpt.py:40]: It has been added here [https://github.com/idea-fasoc/OpenFASOC/pull/270/files#diff-b88c7420f91a0c9ff62c4eb29a049edd0f4ce38053c9539c6dab911338ec2645](url), in PR #270

Comment (saicharan0112) [reply .github/scripts/parse_rpt.py:40]: understood , thanks!

**Comment by saicharan0112** [comment]
@chetanyagoyal : I was worried about this, you have merged conflicts.. next time please try to use branches, it will be really easy to keep everything in a single repo where you can merge branches, maintain different versions temporarily and revert back easily

**Comment by saicharan0112** [comment]
Also please prepare a few graphics to explain how you are checking for @msaligane for tomorrow's meeting

**Comment by chetanyagoyal** [comment]
Closing this PR as it is kind of redundant now, have already included all changes in #270

**Pull request closed without merging**